test/functional/apps/dashboard/dashboard_time_picker.ts

```diff
@@ -76,8 +76,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           'Jan 1, 2000 @ 00:00:00.000',
           'Jan 1, 2000 @ 01:00:00.000'
         );
-        const noResults = await dataGrid.hasNoResults();
-        expect(noResults).to.be.ok();
+        const isLoading = await dataGrid.isLoading();
+        expect(isLoading).to.be(true);
```

Comment:
with the change of my previous comment, this change can be reverted. furthermore this could be flaky in the current implementation. if the the loading state changes to the no results state faster than the check, this could fail.